Source/Public/LevelStatsCollector.h

```diff
@@ -1,9 +1,21 @@
 ﻿#pragma once
 
+#include <ChartCreation.h>
 #include <CoreMinimal.h>
 
 #include "LevelStatsCollector.generated.h"
 
+class MAPMETRICSGENERATION_API FCustomPerformanceChart final : public FPerformanceTrackingChart
```

Comment:
no need for MAPMETRICSGENERATION_API macro since you're not exporting this class to outside modules

Reply:
done.